source/system/Text.ooc

```diff
@@ -172,6 +172,11 @@ Text: cover {
 		this print()
 		t"\n" print()
 	}
+	read: static func (length := 64) -> This {
+		input := CString new(length)
+		fgets(input, length, stdin)
+		This new(input, length + 1)
```

Comment:
`length + 1` because `CString` allocates `length + 1` bytes to make room for `\0`.